src/classes/testAccessController.cls

```diff
@@ -97,14 +135,21 @@ private class testAccessController {
         try {
         	ESAPI.accessController().setOperationMode(null);
         	System.assert(false, 'Should never get here');
-        } catch (Exception e) {
+        } catch (SFDCAccessController.AccessControlDmlException e) {
         	// should fail - so all good
         }
         
         try {
         	ESAPI.accessController().setSharingMode(null);
         	System.assert(false, 'Should never get here');
-        } catch (Exception e) {
+        } catch (SFDCAccessController.AccessControlDmlException e) {
+        	// should fail - so all good
+        }
+
+		try {
```

Comment:
weird indent

Reply:
I've found that VSCode has defaulted to tabs with a size of 4. That's created an odd mixture of the existing 4-space indents and the newer 4-tab indents that doesn't look right on Github.
I did look at converting the entire class to consistent 4-space indenting, but it also picked up a large number of changes on the existing lines. I suspect this was part of the formatting problem I encountered when converting to the SFDX format.

For now I've just corrected this particular instance and will save the consistent document indentation for a subsequent pull request. Or would you prefer it all in one go?

Reply:
Save the whitespace diffs for a different PR please. Formatting changes fit better as their own verifiably formatting-only PR.

Reply:
OK. I think the PR has been corrected now and doesn't include the class wide formatting changes.